index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Landing Page</title>
+
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="/common.css" />
+    <link rel="stylesheet" href="https://main--eager-nightingale-ec3f88.netlify.app/components/component.css">
+</head>
+<body>
+    <div class="container">
+        <nav class="header">
+            <div class="header__btm__left">
+                <div class="logo">
+                    <img src="./images/logo.png" />
+                </div>
+                <ul class="style-none flex-row">
+                    <li class="list-item"><a class="color-white" href="/index.html">Home</a></li>
+                    <li class="list-item"><a class="color-white" href="/product/product.html">Product</a></li>
+                </ul>
+            </div>
+            <div class="search flex-row">
+                <input type="text" class="custom__input">
+                <button class="cs-btn cs-btn-primary">Search</button>
+            </div>
+            <!-- <div class="header__btm__right"></div> -->
+        </nav>
+
+        <div class="main-container">
+            <div class="banner-container component-image-square">
+                <img src="./images/banner.jpeg" alt="banner-image" />
+                <div class="image-overlay-text">
+                    <div class="font-lg padding-1">
+                        Shoppy Collection 2022
+                    </div>
+                    <div class="font-sm padding-2">Go To Place for</div>
+                    <button class="cs-btn cs-btn-primary">
+                        Shop Now<i class="fas fa-arrow-right ml-1"></i>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <!-- ***************************** Brands ended ************************************************** -->
+        <!--
+    **************************** Footer started ************************************************* -->
+
+        <!-- ****************************Footer ended ************************************************ -->
```

Comment:
Edit this part if not done on purpose....

Reply:
removed this